CONTRIBUTING.md

```diff
@@ -0,0 +1,42 @@
+# 📖 Contributing rules in `ufpy` repository
+
+If you want to contribute, you have to read this rules and follow they.
```

Comment:
If you want to contribute **the project**, you **must** read this rules and follow they.